android/res/values-ru/strings.xml

```diff
@@ -834,4 +834,5 @@
 	<string name="error_enter_correct_storey_number">Редактируйте здания высотой максимум %1$s этажей.</string>
 	<string name="editor_zip_code">Почтовый индекс</string>
 	<string name="error_enter_correct_zip_code">Введите корректный почтовый индекс</string>
+	<string name="editor_operator">Оператор</string>
```

Comment:
Как в документе написано - так и написал.
On Thu, Apr 14, 2016 at 3:46 PM Alex Zolotarev <notifications@github.com>
wrote:

> In android/res/values-ru/strings.xml
> <https://github.com/mapsme/omim/pull/2892#discussion_r59710065>:
>
> > @@ -834,4 +834,5 @@
> >  	<string name="error_enter_correct_storey_number">Редактируйте здания высотой максимум %1$s этажей.</string>
> >  	<string name="editor_zip_code">Почтовый индекс</string>
> >  	<string name="error_enter_correct_zip_code">Введите корректный почтовый индекс</string>
> > +	<string name="editor_operator">Оператор</string>
>
> Владелец
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mapsme/omim/pull/2892/files/dccc0daa5f977f509d0c8243231059118cfc8d07#r59710065>
>